languages/elixir/concepts/maps/introduction.md

```diff
@@ -1,4 +1,4 @@
-Maps in Elixir are the goto data structure for storing information in key-value pairs. In other languages, these might also be known as associative arrays (PHP), hashes (Perl 5, Raku), or dictionaries (Python).
+Maps in Elixir are the data structure for storing information in key-value pairs. In other languages, these might also be known as associative arrays (PHP), hashes (Perl 5, Raku), or dictionaries (Python).
```

Comment:
I fear somebody might think we mean "goto" as in "GOTO line 20"... I think the sentence works without it.